src/mruby/get_mruby.sh

```diff
@@ -35,6 +36,9 @@ mkdir -p mruby-out/src/mrbgems
 
 cd mruby-$VERSION
 
+# Use only supported gems.
+cat $DIR/mrusty.gembox > mrbgems/default.gembox
```

Comment:
Needed for `minirake` not to create `_init` / `_final` functions for `mruby-io` and `mruby-bin*`.